xray/core/dataarray.py

```diff
@@ -333,6 +334,11 @@ def values(self):
         """The array's data as a numpy.ndarray"""
         return self.variable.values
 
+    @property
+    def masked_array(self):
```

Comment:
I think I'm in agreement here.  The benefit is cleaner syntax when plotting but I don't think it is required. Will remove.